UPGRADE-5.0.md

```diff
@@ -63,6 +63,7 @@ EventDispatcher
 ---------------
 
  * The `TraceableEventDispatcherInterface` has been removed.
+ * The signature of the `EventDispatcherInterface::dispatch()` method has been updated to `dispatch($event, string $eventName = null)`
```

Comment:
What about `dispatch(/* Event */ $event, string $eventName = null)` to enforce the fact the "new" signature will only accept an `Event` instance ?

In 6.0 we could update it right ?

Reply:
No need: we'll be able to do this in 5.0, since PHP 7.2 will allow adding the type-hint to the interface without breaking BC.